static/templates/partials/iframely-widget-card.tpl

```diff
@@ -10,7 +10,7 @@
 
             <!-- IF title -->
             <h4 class="media-heading">
-                <a href="{embed.url}" target="_blank" rel="nofollow" class="one-line">
+                <a href="{embed.meta.canonical}" target="_blank" rel="nofollow" class="one-line">
```

Comment:
`url` and `meta.canonical` may differ in rare cases when page provides different canonical in its `HEAD`. Example with bit.ly:

    {
        "url": "http://bit.ly/1KOeSFC",
        "meta": {
            "canonical": "https://500px.com/photo/60701250/boy-with-dog-by-jarom%C3%ADr-chalabala"

I think `url` is better for cases when author expects some hits on his shorten url from his post.